src/app/core/entity-components/entity-utils/view-components/readonly-function/readonly-function.component.html

```diff
@@ -0,0 +1 @@
+{{ displayFunction(entity) }}
```

Comment:
we should probably calculate this once onChange in the component and use a property here ... 

https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496